ENGDOCS 711 Revisited Compose Install Page (#14838)
docker/docker.github.io: compose/install/index.md

---
description: How to install Docker Compose
keywords: compose, orchestration, install, installation, docker, documentation
title: Install Docker Compose
toc_max: 2
---

> This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.

Reviewer: I'd tweak this sentence to emphasise more that this page is about checking the installation scenario. It doesn't provide specific instructions on how to install compose. So maybe:
'You can run Compose on macOS, Windows and 64-bit Linux. Use the information on this page to check which installation scenario fits your needs.'

Author: You're right after the split into pages your comment makes sense. In your peer review feel free to always use the suggestion option.

> ## Prerequisites

Reviewer: Wonder if you need these prereqs here and repeated on the compose-plugin page? It implies that these are prereqs for Docker Desktop as well.

Author: OK I will be changing this to Compose Prerequisites to clarify.

* Docker Compose requires Docker Engine.
* Docker Compose plugin requires Docker CLI.

> ## Install Compose

Reviewer: If you've gone with my suggestions above I'd remove this header

Author: I need the anchor, or at least I think I do, lol. So would "Compose installation scenarios (summary)" work for you?

Reviewer: Yes, that would work

Author: PS: Changed it Compose Installation Scenario to address this and your next comment in this conversation.

Check what installation scenario fits your needs. Are you looking to:

> * __Get latest Compose and pre-requirements of the bat__:

Reviewer: And I'd make these top-level bullet points, headings instead.

Author: Why headings? I'm not putting content under it (and I want content to options to be close to each other). I see this as a manual TOC, sort of a description list: <item>:<description>.

Reviewer: I just felt it get's a little lost and provides an entry point for users into the different installation scenarios you have.

Install Docker Desktop for your platform. This is the fastest route and you get Docker Engine and Docker CLI with the Compose plugin.

> Docker Desktop is available for Mac, Windows and Linux.

Author: It is indeed. Thanks!

* __Install Compose plug in:__

>   + __(Mac, Win, Linux) Docker Desktop__: If you got Desktop installed then you're all set, no need to install anything separatedly.

Reviewer: ```suggestion
  + __(Mac, Win, Linux) Docker Desktop__: If you have Docker Desktop installed then you do not need to install anything separately.
```

Author: POV: Thinking that I was going for listing actions, but one could have done this at an earlier stage and taking your suggestion I think something "If you have (already) installed DD..." to work better, yes. Thanks.

Spelling: Thanks. Was thinking about this. Grammarly was my go-to. Is there a spelling extension for VC? If any exist (I'll have look) as well) that we're using as team that you know of at least.

Tone: I find the suggestion to replace "then you're all set" to "then you do not need" is one for a more neutral and dry tone of voice. 
I am aiming to, when writing, to use direct, second-person pronouns (such as “you”) and simple, casual phrases that create a friendly and inviting tone.  I'd like to advocate for that being aligned with our developers mostly informal culture, our visual identity, specifically our mascots.

Author: Thanks for the review @aevesdocker. I will wait to see if have more feedback next week, might commit some of your suggestions on Monday and then ping after tech review or time cut off (?).

Author: PS: Changed this as well.
